projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -234,11 +259,18 @@ export class IgxInputGroupComponent extends DisplayDensityBase
         public element: ElementRef<HTMLElement>,
         @Optional()
         @Inject(DisplayDensityToken)
-        private _displayDensityOptions: IDisplayDensityOptions
+        _displayDensityOptions: IDisplayDensityOptions
     ) {
         super(_displayDensityOptions);
     }
 
+    ngAfterContentInit() {
+        const variant = window
```

Comment:
Direct `window` reference may interfere with server rendering, other uses in the product rely on the injected DOCUMENT w/ `document.defaultView.getComputedStyle`